src/components/Search/Search.tsx

```diff
@@ -43,7 +53,7 @@ const Search = <ObjectType extends TeamsType | UserDataType>(props: SearchProps<
     };
 
     return (
-        <SearchContainer>
+        <SearchContainer fixOnHeader={fixOnHeader} style={style}>
```

Comment:
Allowing custom style and created the prop to fix the search on the header, for the main page